### PR TITLE
feature(webhook-pr):pr webhook support display commit statuses

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 @supereagle 

**Release note**:

```release-note
pr webhook supports display commit statuses
```

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -59,3 +59,33 @@ Environment variables:
 | CALLBACK_URL              | The URL used for webhook to callback, default is http://127.0.0.1:7099/v1/pipelines       |
 | CYCLONE_SERVER            | The host of Cyclone-Server, default is http://localhost:7099. |
 | WORKER_IMAGE              | The image name of Cyclone-Worker container, default is cargo.caicloud.io/caicloud/cyclone-worker:latest. |
+|NOTIFICATION_URL|URL of notification, we will send notification after pipeline execution if the notification policy had been defined.|
+|RECORD_WEB_URL_TEMPLATE|RECORD_WEB_URL_TEMPLATE is a customer's pipeline record website URL address template.|
```
Comment (cd1989) [inline docs/setup.md]: `URL template of pipeline record web page. Cyclone doesn't provide web UI component currently.`

File: pkg/scm/provider/svn/svn.go [hunk 2]
```diff
@@ -121,3 +121,8 @@ func (s *SVN) DeleteWebHook(repoURL string, webHookUrl string) error {
 func (s *SVN) GetTemplateType(repo string) (string, error) {
 	return "", nil
 }
+
+// CreateStatuses generate a new status for repository.
+func (g *SVN) CreateStatuses(state, description, targetURL, repoURL, statusesURL string) error {
+	return nil
```
Comment (cd1989) [inline pkg/scm/provider/svn/svn.go]: Return error to state unsupported.

**Comment by zhujian7** [comment]
all comments have been fixed.

**Pull request merged**